ntc_templates/templates/index

```diff
@@ -264,6 +264,7 @@ cisco_ios_show_capability_feature_routing.textfsm, .*, cisco_ios, sh[[ow]] cap[[
 cisco_ios_show_ip_bgp_vpnv4_all_neighbors.textfsm, .*, cisco_ios, sh[[ow]] ip bgp vpnv4 all nei[[ghbors]]
 cisco_ios_show_ip_eigrp_interfaces_detail.textfsm, .*, cisco_ios, sh[[ow]] ip ei[[grp]] i[[nterfaces]] de[[tail]]
 cisco_ios_show_ip_eigrp_neighbors_detail.textfsm, .*, cisco_ios, sh[[ow]] ip ei[[grp]] nei[[ghbors]] de[[tail]]
+cisco_ios_show_sdwan_control_connections.textfsm, .*, cisco_ios, sh[[ow]] sdw[[an]] cont[[rol]] connections
```

Comment:
@sjhloco
I don't have access to a Cisco Viptela SD-WAN deployment anymore ... this command is the one that has to be spelled out completely on the last word (connections), isn't it? :sweat_smile: